vllm/config.py

```diff
@@ -325,7 +325,8 @@ def _verify_cache_dtype(self) -> None:
         elif self.cache_dtype == "fp8":
             if not is_hip():
                 nvcc_cuda_version = get_nvcc_cuda_version()
-                if nvcc_cuda_version < Version("11.8"):
+                if nvcc_cuda_version is not None \
+                        and nvcc_cuda_version < Version("11.8"):
```

Comment:
Just curious, why is this check on nvcc and not on libcudart version in the first place?
E.g. `torch.version.cuda` or some other way?

---

I was wondering if I have cuda runtime 11.8 without nvcc installed this condition evaluates to False and no error would be raised.

For the docker image it would not matter because it has cuda 12.x

Reply:
It is better to check cuda version on libcudart, as vllm-openai images only has cuda runtime.

Reply:
+1 I fixed a bug for removing nvcc dependency: https://github.com/vllm-project/vllm/issues/4666